docs/current-projects/social-context-alpha.md

```diff
@@ -11,8 +11,29 @@ This study aims to answer the question of: is there an association between socia
 
 ### Background
 
-[insert background on why this question is important, including key citations (for the research community) and Wikipedia/other links (for the public at large)]
+Broadly, this study aims to clarify relations between particular cognitive abilities, personality and social factors, and mental health outcomes such as social anxiety. Prior work suggests that individuals with a temperament known as “behavioral inhibition” are more likely to develop social anxiety if they also exhibit particular profiles of cognitive control ability, including impaired task switching or superior inhibitory control (White et al., 2011; Buzzell et al., 2020). However, it remains unclear how much these associations depend on the context in which task switching and inhibitory control are assessed (i.e. within social settings as opposed to while alone). Additionally, it remains unclear if these associations depend on the temperament of the individual, other personality or social factors, or are present for the majority of individuals with social anxiety. There is at least some work to suggest that the link between some forms of cognitive control (e.g. error monitoring) and social anxiety are influenced by whether cognitive control is assessed within a social context or not (Buzzell et al., 2017). However, there is little work investigating the effect of social context on relations between social anxiety, task switching, and inhibitory control. Moreover, task switching and inhibitory control are related to the ability to maintain and manipulate information within short-term memory, a process referred to as “working memory”, but the link between working memory, social anxiety and temperament is unknown. The current study aims to not only close these gaps in the literature, but also provide a more broad investigation of how particular cognitive abilities, and personality or social factors, relate to mental health outcomes. Towards these ends, participants will perform a task battery that is designed to assess task switching, inhibitory control, and working memory, when responding to either social stimuli (faces) or neutral stimuli (scrambled faces). Additionally, the task battery will be performed twice, once while alone and once while being observed by the experimenter. Task development was based on the modification of existing paradigms and using a standardized set of face stimuli (Lundqvist et. al., 1998). Participants will also complete a more general battery of questionnaires and executive function tasks (flanker task, dimensional change card sort task, n-back task).
+
+Sources for more information on topics related to this research:
```

Comment:
For readability, I would suggest (rather than bold headers and full-text links) to make each of those bold headers into a bullet point and directly add the hyperlink.  For instance:
`* [observational research](https://blahblah)`

Reply:
@jessb0t Done in the new commit!